allensdk/brain_observatory/nwb/nwb_utils.py

```diff
@@ -57,9 +56,11 @@ def set_omitted_stop_time(stimulus_table: pd.DataFrame,
 
 
 def get_image(nwbfile: NWBFile, name: str, module: str) -> Image:
-    nwb_img = nwbfile.modules[module].get_data_interface('images')[name]
```

Comment:
Will this cause problems for people who have already downloaded NWB files? Or is `nwbfile.modules[module]` strictly equivalent to `nwbfile.processing[module]`?

Reply:
The IDE warned me that `nwbfile.modules` was deprecated in favor of `nwbfile.processing`. I believe they are equivalent.